extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -281,14 +279,16 @@ export class InstrumentationService {
   }
 
   /**
-   * Ensure we have a click listener registered on the document.
+   * Ensure we have a click listener registered on the document that contains
+   * the given analytics element.
+   * @param {!Element} analyticsElement
    * @private
    */
-  ensureClickListener_() {
-    if (!this.clickHandlerRegistered_) {
-      this.clickHandlerRegistered_ = true;
-      this.win_.document.documentElement.addEventListener(
-          'click', this.onClick_.bind(this));
+  ensureClickListener_(analyticsElement) {
+    const doc = analyticsElement.ownerDocument;
+    if (!doc[CLICK_LISTENER_REGISTERED_]) {
+      doc[CLICK_LISTENER_REGISTERED_] = true;
```

Comment:
@dvoytenko, @avimehta I am rebasing the other analytics-doc PR #5693, is this still needed in an ampdoc world?